client/src/components/Map.jsx

```diff
@@ -48,7 +50,51 @@ const Map = () => {
       mapboxApiAccessToken={process.env.REACT_APP_MAPBOX_API_KEY}
       dragRotate={false}
       touchRotate={false}
-    />
+    >
+      <div
+        className="searchBar"
+        style={{ position: "absolute", left: 20, top: 20 }}
```

Comment:
not totally sure, but it is possible that you would want to put `role='search'` in this div, then for the aria-labels for input and button inside, you would remove redundant "search" words --> ie) `aria-label="Submit search" `would turn into `aria-label="Submit"`
reference: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/Search_role